testing/README.md

```diff
@@ -98,8 +98,7 @@ gcloud beta builds triggers create github \
 --build-config=cloudbuild.yaml \
 --repo-name=cloud-run-samples \
 --repo-owner=GoogleCloudPlatform \
---pull-request-pattern="^master$"
---name=$TRIGGER_NAME
```

Comment:
"name" returns this error:  `ERROR: (gcloud.beta.builds.triggers.create.github) unrecognized arguments: --name= (did you mean '--repo-name'?) `